src/Compilers/CSharp/Test/Syntax/Parsing/CollectionLiteralParsingTests.cs

```diff
@@ -10609,6 +10609,289 @@ public void MemberAccess22A()
         EOF();
     }
 
+    [Fact]
+    public void MemberAccess23()
```

Comment:
consider naming these tests something like PostFixOperator

Reply:
Will fix in next pr